src/ApiCompat/Microsoft.DotNet.ApiCompatibility/Rules/CannotChangeVisibility.cs

```diff
@@ -24,6 +25,15 @@ public CannotChangeVisibility(RuleSettings settings, IRuleRegistrationContext co
             context.RegisterOnTypeSymbolAction(RunOnTypeSymbol);
         }
 
+        private Accessibility normalizeAccessibility(Accessibility a) =>
+            (_settings.IncludeInternalSymbols, a) switch
+            {
+                (false, Accessibility.ProtectedAndInternal
```

Comment:
Shouldn't `ProtectedAndInternal` and `Internal` be *excluded* when IncludeInternalSymbols is false?  IOW these should be mapped to Private.